### PR TITLE
Update project-lifecycle.md

### PR DESCRIPTION
**Pull request by SecurityCRob**
per 9july tac call, adjustments to archiving process

### CODE REVIEW

**Comment by SecurityCRob** [comment]
per https://github.com/ossf/tac/blob/main/process/TAC-Decision-Process.md, we have a 15day review period and need no less than 7 TAC members to approve (6 + me)

File: process/project-lifecycle.md [hunk 1]
```diff
@@ -159,6 +159,16 @@ When voting on a proposal to archive a project, TAC members may consider:
...
 
+#### Considerations when Archiving a TI that has sub-efforts embedded within ####
+
+1.) Overseeing TI will be moving to "Archived" status
+2.) Active subgroups (SIG or project) is still active and desires to continue collaborating
+3.) Subgroup reviews active TIs to see if there is alignment of vision and purpose (https://openssf.org/community/openssf-working-groups/)
+4.) Subgroup meets with TI leader and group to discuss adoption
+5.) New TI agrees to become new oversight group for subgroup
+6.) PRs filed in TI and TAC repos to note change of status
+7.) Archiving TI has PR filed noting new status
```
Comment (marcelamelara) [inline process/project-lifecycle.md:170]: I like this list, but I might split this into steps/tasks for the hosting TI vs. the sub-TI that wants to transition. I also think that'll also make it clearer about who's responsible for which of these tasks (TAC vs. TI chairs, vs sub-TIs).

File: process/project-lifecycle.md [hunk 2]
```diff
@@ -159,6 +159,16 @@ When voting on a proposal to archive a project, TAC members may consider:
 * If the project's inactivity or inconsistent maintenance presents a user security risk.
 * If the project's design or approach is no longer a recommended best practice.
 
+#### Considerations when Archiving a TI that has sub-efforts embedded within ####
+
+1.) Overseeing TI will be moving to "Archived" status
+2.) Active subgroups (SIG or project) is still active and desires to continue collaborating
```
Comment (marcelamelara) [inline process/project-lifecycle.md:165]: Nit

```suggestion
2.) Whether subgroups/-TIs (SIG or project) are still active and desire to continue collaborating
```

**Review by marcelamelara — COMMENTED** [review]
Thanks @SecurityCRob ! A couple comments...

File: process/project-lifecycle.md [hunk 3]
```diff
@@ -159,6 +159,16 @@ When voting on a proposal to archive a project, TAC members may consider:
...
 
+#### Considerations when Archiving a TI that has sub-efforts embedded within ####
+
+1.) Overseeing TI will be moving to "Archived" status
+2.) Whether subgroups/-TIs (SIG or project) are still active and desire to continue collaborating
+3.) Subgroup reviews active TIs to see if there is alignment of vision and purpose (https://openssf.org/community/openssf-working-groups/)
+4.) Subgroup meets with TI leader and group to discuss adoption
+5.) New TI agrees to become new oversight group for subgroup
+6.) PRs filed in TI and TAC repos to note change of status
+7.) Archiving TI has PR filed noting new status
```
Comment (steiza) [inline process/project-lifecycle.md:170]:
```suggestion
1. Overseeing TI will be moving to "Archived" status
2. Whether subgroups/-TIs (SIG or project) are still active and desire to continue collaborating
3. Subgroup reviews active TIs to see if there is alignment of vision and purpose (https://openssf.org/community/openssf-working-groups/)
4. Subgroup meets with TI leader and group to discuss adoption
5. New TI agrees to become new oversight group for subgroup
6. PRs filed in TI and TAC repos to note change of status
7. Archiving TI has PR filed noting new status
```

Minor: GitHub's Markdown renderer will smoosh all these items onto one line if the format `1.)` is used, but `1.` will create a list 🤷

**Review by steiza — APPROVED** [review]
Minor non-blocking comment

**Review by sevansdell — APPROVED** [review]
Thanks for documenting this Crob. Let's push it forward and iterate as needed.

File: process/project-lifecycle.md (hunk 3 above)
Comment (sudo-bmitch) [reply process/project-lifecycle.md:170]: In addition to removing the `)`, every item can be `1.` and the markdown render will automatically increment. That makes future PRs easier if anything ever gets added or removed from the middle of the list.

**Review by mlieberman85 — APPROVED** [review]

**Review by torgo — APPROVED** [review]
lgtm

**Review by marcelamelara — APPROVED** [review]
Thanks for the updates! LGTM!

File: process/project-lifecycle.md [hunk 4]
```diff
@@ -149,7 +149,7 @@ See [Submission Process](#submission-process) below and [Graduation application]
 
 ### Archived
 
-Open source projects have a lifecycle and there are times when projects become inactive due to a variety of reasons. There are also cases where a project may no longer want to be supported by the OpenSSF, or the OpenSSF TAC may no longer wish to recommend the use of a project. Archiving happens through a vote of the TAC, and can be requested by the corresponding project's lead(s) or a TAC member.
+Open source projects have a lifecycle and there are times when projects become inactive due to a variety of reasons. There are also cases where a project may no longer want to be supported by the OpenSSF, the given effort no longer has broad community interest and particpation, or the OpenSSF TAC may no longer wish to recommend the use of a project. Archiving happens through a vote of the TAC, and can be requested by the corresponding project's lead(s) or a TAC member.  TI's that are dormant, with no activity for 9 months (meetings, mailing lists, or other publicly viewable channels) in a row should be considered good candidates for Archiving.
```
Comment (lehors) [inline process/project-lifecycle.md:152]:
```suggestion
Open source projects have a lifecycle and there are times when projects become inactive due to a variety of reasons. There are also cases where a project may no longer want to be supported by the OpenSSF, the given effort no longer has broad community interest and participation, or the OpenSSF TAC may no longer wish to recommend the use of a project. Archiving happens through a vote of the TAC, and can be requested by the corresponding project's lead(s) or a TAC member.  TI's that are dormant, with no activity for 9 months (meetings, mailing lists, or other publicly viewable channels) in a row should be considered good candidates for Archiving.
```

**Review by lehors — APPROVED** [review]

**Pull request merged**